agent/bench-scripts/pbench-fio

```diff
@@ -115,8 +115,10 @@ function usage {
 	printf "\t\tfile sizes in MiB\n"
 	printf "\n"
 	printf -- "\t-d str[,str] --targets=str[,str]\n"
-	printf "\t\tone or more directories or block devices (default is $targets)\n"
-	printf "\t\t(persistent names for devices highly recommended)\n"
+	printf "\t\tone or more files, block devices, or directories (default is $targets)\n"
+	printf "\t\tpersistent names for block devices highly recommended; to use a directory,\n"
+	printf "\t\tuse the ${script_path}/templates/fio-shared-fs.job file or specify a\n"
+	printf "\t\tjob file with --job-file=<path> which uses the 'directory = \$target' syntax\n"
```

Comment:
The pseudo BNF (i.e., `[,str]`) doesn't make it clear that more than two items can be specified.  Perhaps adding an elipsis would suffice ((i.e., `[,str...]` or `[,str]...` or `[[,str]...]]`).

The phrase `persistent names for block devices highly recommended` is sort of floating in space.  It should be preceded by a comma, em-dash, or semicolon, and it might be good if it were placed inside the parenthetical.

The clause starting with `to use a directory` is confusing:  it seems to contradict the preceding verbiage which explains how to specify a directory on the command line.  Perhaps we could add some words which make it clear that these are alternative means for specifying the directory.

Reply:
I addressed the elipsis change as a separate commit since it really is not part of this fix.

Hopefully the revised text is clear.